src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -56,14 +60,11 @@ def __init__(
         self._migrate_grants = migrate_grants
         self._external_locations = external_locations
 
-    def get_remaining_tables(self) -> list[Table]:
-        migration_index = self.index(force_refresh=True)
-        table_rows = []
+    def check_remaining_tables(self, migration_status: Iterable[TableMigrationStatus]) -> None:
```

Comment:
Plural for an iterable

```suggestion
    def check_remaining_tables(self, migration_statuses: Iterable[TableMigrationStatus]) -> None:
```

Reply:
And maybe rename it to `warn_about_remaining_tables`. I expect a `True` / `False` gate-like logic when the name is `check_`, though `check_` is also accurate to an extend

Reply:
> And maybe rename it to `warn_about_remaining_tables`. I expect a `True` / `False` gate-like logic when the name is `check_`, though `check_` is also accurate to an extend

I've gone with the `warn_about_[…]` suggest: it's closer to what it does.

Reply:
Interesting, I use statuses: https://grammarhow.com/plural-of-status/. Still statuses is more clear to be plural than status, as status can be singular too